<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTAGenestaImportWidget</class>
 <widget class="QDialog" name="GTAGenestaImportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>261</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import Genesta library</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/images/forms/img/GTAAppIcon.png</normaloff>:/images/forms/img/GTAAppIcon.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="libFileFolLB">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Genesta procedure:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="GenestaFilePathLE">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BrowseLibFolderPB">
       <property name="toolTip">
        <string>Browse folder</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/images/forms/img/OpenFolder.png</normaloff>:/images/forms/img/OpenFolder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Genesta object:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="libObjectPathLE">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="browseObjectFolderPB">
       <property name="toolTip">
        <string>Browse folder</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/images/forms/img/OpenFolder.png</normaloff>:/images/forms/img/OpenFolder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Export folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="exportFolderLE">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="browseExportFolderPB">
       <property name="toolTip">
        <string>Browse folder</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/images/forms/img/OpenFolder.png</normaloff>:/images/forms/img/OpenFolder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="errorLogLB">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Error Log:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ErrorlLogFilePathLE">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="BrowseErrorFolderPB">
       <property name="toolTip">
        <string>Browse folder</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/images/forms/img/OpenFolder.png</normaloff>:/images/forms/img/OpenFolder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="replaceFilesCB">
       <property name="text">
        <string>Replace existing files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="openInGTARB">
       <property name="text">
        <string>Open in GTA</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="importFolderRB">
       <property name="text">
        <string>Import folder</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="cancelPB">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/images/forms/img/CANCEL.jpg</normaloff>:/images/forms/img/CANCEL.jpg</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="okPB">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/images/forms/img/OK.png</normaloff>:/images/forms/img/OK.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>GenestaFilePathLE</tabstop>
  <tabstop>BrowseLibFolderPB</tabstop>
  <tabstop>libObjectPathLE</tabstop>
  <tabstop>browseObjectFolderPB</tabstop>
  <tabstop>exportFolderLE</tabstop>
  <tabstop>browseExportFolderPB</tabstop>
  <tabstop>ErrorlLogFilePathLE</tabstop>
  <tabstop>BrowseErrorFolderPB</tabstop>
  <tabstop>replaceFilesCB</tabstop>
  <tabstop>openInGTARB</tabstop>
  <tabstop>importFolderRB</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>cancelPB</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>importFolderRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>GenestaFilePathLE</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openInGTARB</sender>
   <signal>toggled(bool)</signal>
   <receiver>GenestaFilePathLE</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openInGTARB</sender>
   <signal>toggled(bool)</signal>
   <receiver>libObjectPathLE</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importFolderRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>libObjectPathLE</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
